### PR TITLE
Rewrote TTestIndependentSamples with jaspResults

### PR DESCRIPTION
**Pull request by Janto91**
I also added more effect sizes and changed the computation of them via packages (after consultation with E.J. and Simon).
I also added an option for the user to determine which standard deviation should be used for computing Cohen's d and Hedges' g.

### CODE REVIEW

**Comment by Janto91** [comment]
@Kucharssim: Could you take a look at unit tests for the effect sizes? I compared the results for Cohen's d to the ones reported in Field (Chapter 4) but, so far, we do not have unit tests for Hedges' g and Cliff's delta and the corresponding confidence intervals.

File: JASP-Engine/JASP/R/ttestindependentsamples.R [hunk 1]
```diff
@@ -15,822 +15,857 @@
...
+                                                           type = "number",          format = "sf:4;dp:3",
+                                                           overtitle = paste0(100*options$effectSizeConfidenceIntervalPercent, "% CI for Hedges' g"))
+    }
+  }
+  
+  # Fill up table with results
+  .fillUpIndependentSamplesTTestTable(independentSamplesTTestTable = independentSamplesTTestTableParametric, 
+                                      dataset = dataset, options = options, parametric = TRUE, ready = ready)
+  
+  return(NULL)
```
Comment (koenderks) [inline JASP-Engine/JASP/R/ttestindependentsamples.R]: I don't think you have to return anything in these jaspResult functions

File: JASP-Engine/JASP/R/ttestindependentsamples.R [hunk 2]
```diff
@@ -15,822 +15,857 @@
...
+                                                              type = "number",             format = "sf:4;dp:3",
+                                                              overtitle = paste0(100*options$effectSizeConfidenceIntervalPercent, "% CI for Cliff's Delta"))
+    }
+  }
+  
+  # Fill up table with results
+  .fillUpIndependentSamplesTTestTable(independentSamplesTTestTable = independentSamplesTTestTableNonParametric, 
+                                      dataset = dataset, options = options, parametric = FALSE, ready = ready)
+  
+  return(NULL)
```
Comment (koenderks) [inline JASP-Engine/JASP/R/ttestindependentsamples.R]: same here (and for all other functions)

**Comment by JohnnyDoorn** [comment]
What's the status on this? I would like to add some stuff to the t-test but want to wait for this rewrite.

**Comment by Janto91** [comment]
I have rewritten this with jaspResults but not following the newest template. I can try to do this on Sunday.

**Pull request closed without merging**